incubator/virtualcluster/cmd/manager/main.go

```diff
@@ -100,6 +102,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	// start a health http server.
+	log.Info("Starting a health http server")
+	mux := http.NewServeMux()
+	healthz.InstallHandler(mux)
+	if err := http.ListenAndServe(":8080", mux); err != nil {
+		log.Error(err, "unable to start health http server")
+		os.Exit(1)
+	}
```

Comment:
I believe we need to wrap this entire thing in a goroutine so that the rest of the controller can start.

Reply:
Good catch, `ListenAndServe` is block call and should be put in go routine